File: Python/Niveau 1/8 – Répétitions conditionnées/Département de chimie mélange explosif.py
tests = int(input())
tmin = int(input())
tmax = int(input())

while True:
   try:
      testss = int(input())
   except EOFError:
      break
   if testss >= tmin and testss <= tmax:
      print("Rien à signaler")
   else:
      print("Alerte !!")
      break

File: Pascal/Niveau 1/8 – Répétitions conditionnées/Département de chimie mélange explosif.pas
program Solution;
var
    mesures, tempMin, tempMax, mesure, temp: LongInt;
    validTemp: Boolean;
begin
    read(mesures, tempMin, tempMax);
    mesure := 0;
    validTemp := true;
    while (mesure < mesures) and (validTemp) do
    begin
        read(temp);
        validTemp := (tempMin <= temp) and (temp <= tempMax);
        if validTemp then
        begin
            writeln('Rien à signaler');
        end
        else
        begin
            writeln('Alerte !!');
        end;
    mesure := mesure + 1;
    end;
end.